custom/conf/app.ini.sample

```diff
@@ -648,18 +646,6 @@ ITEM_TTL = 16h
 
 ; Last commit cache
 [cache.last_commit]
-; if the cache enabled
```

Comment:
@lafriks can wee keep ENABLED for main cache and cache.last_commit section?

Reply:
it duplicates TTL but we can

Reply:
@6543 Added back